src/miner.cpp

```diff
@@ -391,7 +428,7 @@ void BlockAssembler::addPackageTxs(int &nPackagesSelected, int &nDescendantsUpda
 
         if (packageFees < blockMinFeeRate.GetFee(packageSize)) {
             // Everything else we might consider has a lower fee rate
-            return;
+            break;
```

Comment:
As I understand it, this is a miner's feature. We're reading in transactions in order of highest fee density to lowest fee density. You can configure the minimum fee per byte, and if we go below that we're done. Miners want the privilege of being able to ignore transactions below a certain fee; it's to their advantage to hash less data. This will become more important as the mined block payment reduces over time.

Reply:
Ah, i see get it by ancestor_score it will be from high to low, so after that all other txs are with lowest fee - then ok, we can keep return as well.